### PR TITLE
PD-JS1-W1

### PR DESCRIPTION
**Pull request by esma-g**
## What does this change?

Module: JS1
Week(s): 1

## Description

- Learning objectives deleted
- PD session content and coursework added.
- Coursework formatted with numbers
- Session objective added

## Who needs to know about this?

@kfklein15 @ElizabethlawalCYF @SallyMcGrath @CodeYourFuture/syllabus

### CODE REVIEW

**Review by kfklein15 — DISMISSED** [review]
LGTM

**Comment by esma-g** [comment]
> LGTM

This PR is still a draft.

**Review by kfklein15 — DISMISSED** [review]
lgtm

File: docs/js-core-1/week-1/lesson.md [hunk 1]
```diff
@@ -32,6 +33,7 @@ By the end of this class, you should be able to:
 - Call functions within functions
 - Search and read documentation to help when you are stuck
 - Use the Code Style Guide to give feedback on other trainees work
+- Understand the core concepts of the agile software development methodology
```
Comment (SallyMcGrath) [inline docs/js-core-1/week-1/lesson.md]:
```suggestion
- Describe four core concepts of the Agile software development methodology
```

Understand is an overarching term. In a learning objective, choose an active verb that belongs to the understanding group. Bloom's taxonomy may help you here: https://bloomstaxonomy.net/

File: docs/js-core-1/week-1/lesson.md [hunk 2]
```diff
@@ -528,6 +530,14 @@ Using what you've just read in the Style Guide.
 - Script: a file that contains a program
 - Terminal: another word for "console"
 
+## PD (Agile development)
+
+**Session objective:** This session provides an overview of the agile software development methodology.
+
+[Agile development slides](https://docs.google.com/presentation/d/1SN_NIXeKi4-WWWVA_ffxKyc4ZIe5Tem2zcfoYA4RtB4/edit#slide=id.g1adb8e69ab9_0_5)
+
+Notes for the facilitator: Remember to adapt/update the content in line with your cohort’s needs and arrange timings as needed.
```
Comment (SallyMcGrath) [inline docs/js-core-1/week-1/lesson.md]:
```suggestion

```
Please put this in the instructor notes https://syllabus.codeyourfuture.io/js-core-1/week-1/instructors

File: docs/js-core-1/week-1/lesson.md [hunk 3]
```diff
@@ -528,6 +530,14 @@ Using what you've just read in the Style Guide.
 - Script: a file that contains a program
 - Terminal: another word for "console"
 
+## PD (Agile development)
+
+**Session objective:** This session provides an overview of the agile software development methodology.
```
Comment (SallyMcGrath) [inline docs/js-core-1/week-1/lesson.md]:
```suggestion
**Session objective:** This session provides an overview of the Agile software development methodology.
```
Agile is a proper noun.

**Review by SallyMcGrath — DISMISSED** [review]
A few tiny notes, still working on those learning objectives. Here's a video on our teacher training site that you might find helpful:

https://teachertraining.codeyourfuture.io/content/what-is-a-good-learning-objective

**Review by kfklein15 — DISMISSED** [review]
Must have changes have been done for now to bring PD content into the Syllabus page.
Bigger changes will be done with the PD Syllabus review.

**Comment by esma-g** [comment]
Deleting PD learning objectives and not adding any new ones until they are re-created by the PD syllabus review.

File: docs/js-core-1/week-1/lesson.md (hunk 1 above)
Comment (esma-g) [reply docs/js-core-1/week-1/lesson.md]: Deleting PD learning objectives and not adding any new ones until they are re-created by the PD syllabus review.

**Review by kfklein15 — DISMISSED** [review]
LGTM

**Review by illicitonion — DISMISSED** [review]

**Review by illicitonion — APPROVED** [review]

**Pull request merged**